nest/02.introduction.md

```diff
@@ -0,0 +1,51 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all the files required to install, uninstall and update a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage packages**.
+It is similar to an app store for a smartphone, or the Windows Store for example.
+One of the key differences between those and Nest (and most package managers), is that the upgrading must be done manually.
+On a smartphone, the applications are usually automatically upgraded.
+With Nest, it's up to the user to upgrade their installed packages whenever they want.
+
+Currently, to use Nest, one must use its Command Line Interface (CLI).
+This might sound intimidating for someone who has never seen a terminal or typed any command before, but if that's the case, don't worry! This guide is here to help you feel at home.
+
+## 1.2 Packages and dependencies
+It is common for a package to require other ones to work properly.
+For example, let's take a software that allows its users to send messages to each other, over the Internet.
+That software will need a way to send messages over the Internet.
+To do so, it will either implement its own solution, or use an existing library for that purpose.
+In the latter case, the software **requires** that library, because without it, the messages simply can't be sent.
+In other words, that library is a **dependency** of the software.
+
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+If it does, Nest will install the dependencies that aren't already installed.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
```

Comment:
Does this sentence have any purpose ? I don't think it conveys any information.
Its tone is also a bit weird.

Reply:
Its goal is to explain that when you install a package, you can see other packages getting installed too, which are its dependencies
If I'm totally new to Nest and package managers in general, I'd be surprised if when I'm installing a single package, other packages are installed with it. Even if after reading this section you should know that they are dependencies, I figured it's not that big of a deal to explicitly write it

Reply:
Hum, I'd change the sentence above then.

Reply:
How about this?
```suggestion
In other words, when asking Nest to install a package, other packages will be installed alongside it to satisfy it's dependencies.
```
This way it's obvious you're restating the same thing than the previous statement and we keep a neutral tone.